.gitignore

```diff
@@ -22,6 +22,13 @@ __pycache__/
 
 # Generated files
 stash
+downloaded_granules
```

Comment:
`download_granules` saves a file into the current directory, so I added these file types to the `.gitignore`